### ISSUE TITLE
import empty file

### ISSUE THREAD

**Issue by someoneHere** [vusion/icon-font-loader#30]
index.js
`import './style.less'`
style.less
`.a{}`

in this condition,it will get an error:
```
/node_modules/icon-font-loader/src/Plugin.js:149
  } else if (typeof source === 'object' && typeof source._value === 'string') {

TypeError: Cannot read property '_value' of null
````
because the value of source is null

**Comment by AlfredMou** [comment]
From the performance point of view, it should be that when the less-loader handles the introduction of the module, the empty file of .aa{} is processed into a null character, and the source is turned into null here. However, there is a loophole in the type of the source. I will correct this bug later.

**Comment by AlfredMou** [comment]
This was fixed long ago in commit 1f0df7678c82141cafedf2ed9c1e141a60ba4f63
